### PR TITLE
EGKB ILS

### PR DESCRIPTION
**Pull request by ewanfleming**
EGKB ILS fixed.

### CODE REVIEW

File: LOC.json [hunk 1]
```diff
@@ -4023,12 +4023,12 @@
...
     "associatedRunwayNumber": "21",
-    "latitude": 51.32336044,
-    "longitude": 0.02660600,
+    "latitude": 51.33824862,
+    "longitude": 0.03805548,
     "elevation": 598,
     "frequency": 109350,
     "receptionRange": 18,
-    "bearing": 205.647
+    "bearing": 25.65
```
Comment (ItsW1ll) [inline LOC.json]: Add 180 to the bearing to get 205.65 and stick that there

File: Glideslope.json [hunk 2]
```diff
@@ -4073,13 +4073,13 @@
...
-    "latitude": 51.33702087,
-    "longitude": 0.03503300,
+    "latitude": 51.33824862,
+    "longitude": 0.03805548,
     "elevation": 598,
     "frequency": 109350,
     "receptionRange": 10,
-    "bearing": 206.062,
-    "glideslope": 3.00205531
+    "bearing": 25.65,
```
Comment (ItsW1ll) [inline Glideslope.json]: Add 180 to the bearing to get 205.65 and stick that there

File: Glideslope.json [hunk 3]
```diff
@@ -4073,13 +4073,13 @@
...
+    "glideslope": 3.00256500
```
Comment (ItsW1ll) [inline Glideslope.json]: change the glideslope to 3.00205650

**Comment by dushyants19** [comment]
Ewan do you even know Nav

**Comment by ewanfleming** [comment]
Dushywushy do you even know social skills

**Comment by ItsW1ll** [comment]
Let him off a bit.. it’s his first ILS!

**Comment by ewanfleming** [comment]
Committed a fix, should be fine now.

**Pull request merged**